constants/extraRpcs.json

```diff
@@ -917,6 +917,11 @@
             "https://seed.btcix.org/rpc"
         ]
     },

```

Comment:
constants/extrarpcs.json

Reply:
"19845": {
        "rpcs": [
            "https://seed.btcix.org/rpc"
        ]
    },
    **"21004": {
        "rpcs": [
            "https://rpc.c4ei.net"
        ]
    },**
    "21816": {
        "rpcs": [
            "https://seed.omlira.com"
        ]
    },


i added c4ei 19845 next **21004** next 21816

Reply:
"19845": {
        "rpcs": [
            "https://seed.btcix.org/rpc"
        ]
    },
    "21004": {
        "rpcs": [
            "https://rpc.c4ei.net"
        ]
    },

19845 (https://seed.btcix.org/rpc) next object

Reply:
i saw file all is right ^^; thank you

    "19845": {
        "rpcs": [
            "https://seed.btcix.org/rpc"
        ]
    },
    "21004": {
        "rpcs": [
            "https://rpc.c4ei.net"
        ]
    },